salt/modules/yumpkg.py

```diff
@@ -1056,9 +1056,10 @@ def install(name=None,
     return ret
 
 
-def upgrade(refresh=True, fromrepo=None, skip_verify=False, **kwargs):
+def upgrade(name=None, refresh=True, fromrepo=None, skip_verify=False, pkgs=None, normalize=True, **kwargs):
```

Comment:
What kind of problems it can bring? What is the correct way to implement it? Maybe putting this feature in release notes can minimize surprises of users? I tried to move name=None just before pkgs=None, but then
```bash
salt-call pkg.upgrade pkg-needing-upgrade
```
didn't work.